specification/hdinsight/resource-manager/Microsoft.HDInsight/stable/2018-06-01-preview/cluster.json

```diff
@@ -1009,6 +1058,14 @@
         "msiResourceId": {
           "type": "string",
           "description": "The managed identity (MSI) that is allowed to access the storage account, only to be specified for Azure Data Lake Storage Gen 2."
+        },
+        "saskey": {
```

Comment:
The same with `VMGroupName`, per my understanding, `sasKey` will be better, but CI does not report this is not CamelCase, there is just a unknown word issue.